### PR TITLE
message displayed when user do login, solves issue #24

### PR DESCRIPTION
**Pull request by amit6663**

### CODE REVIEW

**Comment by amit6663** [comment]
pls review @therajanmaurya @naman14

**Comment by rachittt** [comment]
@amit6663 pls squash your commits

File: mifos-passcode/src/main/java/com/mifos/mobile/passcode/MifosPassCodeActivity.java [hunk 1]
```diff
@@ -222,8 +224,12 @@ public void passCodeEntered(String passcode) {
                     resetPasscode();
                     return;
                 }
-                Toast.makeText(this, "Login Successfully",
-                        Toast.LENGTH_SHORT).show();
+
+//                Toast.makeText(this, "Login Successfully",Toast.LENGTH_SHORT).show();
+                String info="Login Successfully";
```
Comment (jawidMuhammadi) [inline mifos-passcode/src/main/java/com/mifos/mobile/passcode/MifosPassCodeActivity.java]: Please don't use hard coded strings. First, add in `strings.xml` file then use it. As I see the string has been used as hardcoded before, but please you don't do that.

**Comment by robustTechie** [comment]
Please squash it and also provide description.

**Pull request closed without merging**